test/cmd/images.sh

```diff
@@ -72,7 +72,7 @@ os::cmd::expect_success_and_text 'oc describe istag/mysql:latest' 'Image Created
 os::cmd::expect_success_and_text 'oc describe istag/mysql:latest' 'Image Name:'
 name=$(oc get istag/mysql:latest --template='{{ .image.metadata.name }}')
 imagename="isimage/mysql@${name:0:15}"
-os::cmd::expect_success 'oc describe "${imagename}"'
+os::cmd::expect_success "oc describe ${imagename}"
```

Comment:
@deads2k I think these two should be separate, as this PR is finished and ready as it is, the README has a long way to go yet. This PR is immediately useful and the `local` changes are potential bugs - something I'd rather go in now and not wait for the README.